src/plugins/intel_cpu/src/emitters/jit_snippets_emitters.cpp

```diff
@@ -286,7 +295,7 @@ void KernelEmitter::init_data_pointers(size_t num_inputs, size_t num_params,
             h->mov(data_ptr_regs[i], h->ptr[reg_const_params + GET_OFF(dst_ptrs) + (i - num_inputs) * sizeof(void*)]);
         init_ptr_with_offset(data_ptr_regs[i], data_offsets[i], reg_tmp);
     }
-    // a rare case when num_params is maximal, so we have no spare gprs
+    // a rare case when num_io is maximal, so we have no spare gprs
```

Comment:
It's not clear what `num_io` is. If you want to rename `num_params` to `num_io` - it's fine, but it's better to keep comments consistent with the code.

Reply:
Good catch. As I remember, I renamed to `num_io` but you suggested to leave it so. I removed my changes but forgot about comment.
Reverted, thanks